doc/fitting/fitting.md

```diff
@@ -0,0 +1,201 @@
+# Fitting Callback
+
+Similar to [`LivePlot`](../callbacks/plotting.md), `ibex_bluesky_core` provides a thin wrapper around Bluesky's LiveFit class, enhancing it with additional functionality to better support real-time data fitting. This wrapper not only offers a wide selection of models to fit your data on, but also introduces guess generation for fit parameters. As new data points are acquired, the wrapper refines these guesses dynamically, improving the accuracy of the fit with each additional piece of data, allowing for more efficient and adaptive real-time fitting workflows.
+
+In order to use the wrapper, import `LiveFit` from `ibex_bluesky_core` rather than 
+`bluesky` directly:
+```py
+from ibex_bluesky_core.callbacks.plotting import LiveFit
+```
+**Note:** that you do not *need* `LivePlot` for `LiveFit` to work but it may be useful to know visaully how well the model fits to the raw data.
+
+## Configuration
+
+Below is a full example showing how to use standard `matplotlib` & `bluesky` functionality to apply fitting to a scan, using LivePlot and LiveFit. The fitting callback is set to expect data to take the form of a gaussian.
+```py
+import matplotlib.pyplot as plt
+from ibex_bluesky_core.callbacks.plotting import LivePlot, LiveFit
+
+# Create a new figure to plot onto.
+plt.figure()
+# Make a new set of axes on that figure
+ax = plt.gca() 
+# ax is shared by fit_callback and plot_callback 
+
+plot_callback = LivePlot(y="y_variable", x="x_variable", ax=ax)
+fit_callback = LiveFit(Gaussian.fit(), y="y_variable", x="x_variable", update_every=0.5)
+# update_every = in seconds, how often to recompute the fit. If `None`, do not compute until the end. Default is 1.
+fit_plot_callback = LiveFitPlot(fit_callback, ax=ax, color="r")
```

Comment:
Show the import of LiveFitPlot with the others to make it clear that this is from bluesky not ibex bluesky